[ADD] project_categ_model

martintorresvm opened the pull request:
Add constrain and required field for categ_ids
	modified:   project_categ_model.py

martintorresvm commented:
FYI @elicoidal

martintorresvm commented:
Please review @elicoidal @dreispt @seb-elico

elicoidal commented:
@victormartinelicocorp I think we should be able to have multiple tags.
What we need to remove is the mandatory attribute in the field tag.
For BR management, besides we need a specific field Category (m2o) on top of the tags linked to the same model and allowing only to display a subset.

dreispt commented:
I tend to agree with Eric.

martintorresvm commented:
Ok then I just remove the attribute required. And later I'll will propose a simple module to manage many2one categs.

Could you check again @elicoidal @dreispt ?

martintorresvm commented:
> Ok then I just remove the attribute required. And later I'll will propose a simple module to manage many2one categs.
Could you check again @elicoidal @dreispt ?

@elicoidal I think this is done could you help to merge it?

dreispt reviewed (commented):
Sorry @victormartinelicocorp , but this module is not doing any relevant changes to the module.

elicoidal commented:
Lol indeed...
Somewhere we are lost in translation

martintorresvm commented:
> @victormartinelicocorp I think we should be able to have multiple tags.
**What we need to remove is the mandatory attribute in the field tag.**
For BR management, besides we need a specific field Category (m2o) on top of the tags linked to the same model and allowing only to display a subset.

@elicoidal @dreispt If the point about **remove mandatory** not necessary I rather close this PR

Thanks for review

dreispt commented:
@victormartinelicocorp I think you want to add the required attribute on the Task view, like you did to Issues on another PR.

elicoidal commented:
I just need this "What we need to remove is the mandatory attribute in the field tag."
as we discussed long time ago.
Besides in task, add a specific field m2o linked to the category tags model

dreispt commented:
I'm OK to remove the mandatory.
@victormartinelicocorp can we have a clean diff with just that change?

martintorresvm commented:
> I just need this "What we need to remove is the mandatory attribute in the field tag."
as we discussed long time ago.
Besides in task, add a specific field m2o linked to the category tags model

> I'm OK to remove the mandatory.
@victormartinelicocorp can we have a clean diff with just that change?

Done, please have a check. @elicoidal @dreispt

dreispt commented on project_categ/project_categ_view.xml:
```diff
@@ -44,10 +44,6 @@
-                <field name="categ_ids"  position="attributes">
-                    <attribute name="attrs">{'required':[('task_categ_id','!=',False)]}</attribute>
-                </field>
-
```
I expected this to be the only diff, no Python code changes.

Commit:
[IMP] project_categ
Remove conditional attribute to make required categ_ids
	modified:   project_categ/project_categ_view.xml

martintorresvm commented:
> I expected this to be the only diff, no Python code changes.

Sorry by that I made a mistake with Git reset.  Now I double check after commit and hope it's fine.

cc @elicoidal @dreispt
